airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.py

```diff
@@ -60,27 +81,39 @@ class Config:
         title="End Date",
         pattern="^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$|^$",
         examples=["2017-01-25T00:00:00Z"],
+        order=9,
     )
 
     period_in_days: int = Field(
         30,
         description="Will be used for stream slicing for initial full_refresh sync when no updated state is present for reports that support sliced incremental sync.",
         examples=["30", "365"],
+        order=10,
     )
     report_options: str = Field(
         None,
         description="Additional information passed to reports. This varies by report type. Must be a valid json string.",
-        examples=['{"GET_BRAND_ANALYTICS_SEARCH_TERMS_REPORT": {"reportPeriod": "WEEK"}}', '{"GET_SOME_REPORT": {"custom": "true"}}'],
+        examples=['{"GET_BRAND_ANALYTICS_SEARCH_TERMS_REPORT": {"reportPeriod": "WEEK"}}',
+                  '{"GET_SOME_REPORT": {"custom": "true"}}'],
+        order=11,
     )
     max_wait_seconds: int = Field(
         500,
         title="Max wait time for reports (in seconds)",
         description="Sometimes report can take up to 30 minutes to generate. This will set the limit for how long to wait for a successful report.",
         examples=["500", "1980"],
+        order=12,
     )
 
-    aws_environment: AWSEnvironment = Field(description="Select the AWS Environment.", title="AWS Environment")
-    region: AWSRegion = Field(description="Select the AWS Region.", title="AWS Region")
+    aws_environment: AWSEnvironment = Field(
+        description="Select the AWS Environment.",
+        title="AWS Environment",
```

Comment:
We are not able to add order to `aws_environment` because it is being overridden in https://github.com/airbytehq/airbyte/blob/f57fcdf87cf2572758e1a3dbb3dac5faa93d6609/airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py#L222.